lib/Backend/Lower.cpp

```diff
@@ -3745,10 +3755,12 @@ Lowerer::GenerateProfiledNewScArrayFastPath(IR::Instr *instr, Js::ArrayCallSiteI
         const IR::AutoReuseOpnd autoReuseHeadOpnd(headOpnd, func);
 
         GenerateMemInit(dstOpnd, Js::JavascriptNativeIntArray::GetOffsetOfWeakFuncRef(), IR::AddrOpnd::New(weakFuncRef, IR::AddrOpndKindDynamicFunctionBodyWeakRef, m_func), instr, isZeroed);
+
+        IR::Opnd * missingItemOpnd = getMissingItemOpnd(TyInt32);
         for (; i < size; i++)
         {
             GenerateMemInit(headOpnd, sizeof(Js::SparseArraySegmentBase) + i * sizeof(int32),
-                Js::JavascriptNativeIntArray::MissingItem, instr, isZeroed);
+                missingItemOpnd, instr, isZeroed);
```

Comment:
Do we not need an autoReuseMissingItemOpnd here?

Reply:
I added the others in response to failures. It seemed like it wasn't needed here, but it wouldn't hurt to have. Either way, it's included in the refactor in my commit.